01week/rockPaperScissors.js

```diff
@@ -7,14 +7,55 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
+const normalizeHand = hand => {
+  if (typeof hand !== 'string')
+    return ''; // will be treated as invalid input
 
-function rockPaperScissors(hand1, hand2) {
+  return hand.toLowerCase().trim();  
+}
+
+const isHandValid = hand => {
```

Comment:
Really awesome work! You're doing some pretty advanced stuff here. Nice use of functional programming to compartmentalize the parts of the app.